extensions/sql-migration/src/api/utils.ts

```diff
@@ -162,19 +162,23 @@ export function filterMigrations(databaseMigrations: azure.DatabaseMigration[],
 			return filteredMigration.filter(
 				value => {
 					const status = getMigrationStatus(value);
-					return status === constants.MigrationStatus.InProgress
-						|| status === constants.MigrationStatus.Retriable
-						|| status === constants.MigrationStatus.Creating;
+					return status === constants.MigrationState.InProgress
+						|| status === constants.MigrationState.ReadyForCutover
```

Comment:
are there other icons available? I think it would be nice to have a distinct icon for ReadyForCutover and Retriable states, since those will indicate user action is needed

Reply:
@brian-harris, do we have a UI designer we could reach out to in order to get new icons designed?  Or perhaps we could reuse another one of the existing icons for these states?

Reply:
yep, will do, thanks